bundles/org.openhab.binding.rotel/README.md

```diff
@@ -22,7 +22,7 @@ You can connect it for example to a Raspberry Pi and use [ser2net Linux tool](ht
 Recent devices provide a network interface (RJ45 connector).
 So you can use the device IP to connect to the device, keeping 9590 as default port.
 
-The binding has been tested with a RSP-1066 and a RSP-1570.
+The binding has been tested with a RSP-1066, a RSP-1570 and a RX-1052.
```

Comment:
```suggestion
The binding has been tested with an RSP-1066, an RSP-1570 and an RX-1052.
```

Reply:
Interesting, until today, I thought I must use "an" when the next world starts with a vowel and "a" when consonant !

Reply:
English is only my second language, but I'm pretty sure it's the **sound** of a consonant or vowel. For example "an hour", because you don't pronounce the h. Here it's the letter R, which is pronounced like a vowel, I believe.

Reply:
@andrewfg - since English is your native language, perhaps you can help by proofreading this sentence, since I'm now suddenly in doubt if "R" is to be considered pronounced as vowel or consonant. 😃 I am now hoping that I didn't ask @lolodomo to introduce a grammar mistake.

Reply:
I'm not a native speaker either, but I'm pretty sure you're right, and experts agree with it: https://www.merriam-webster.com/grammar/is-it-a-or-an ... and the pronunciation here clearly is 'ar-ess-pi' and 'ar-ex' here.

Reply:
Thanks @maniac103, I now feel convinced again. 🙂